<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SandraArun" failures="4" tests="18" name="testScripts.LoginPageTest" time="71.265" errors="0" timestamp="2025-01-09T08:59:28 IST" skipped="8">
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="4.698">
    <skipped/>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyInvalidUserNameforLogin" time="4.681"/>
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="3.291"/>
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyInvalidPasswordforLogin" time="4.866"/>
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="4.546">
    <failure message="Dashboard is not displayed expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Dashboard is not displayed expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at testScripts.LoginPageTest.verifyValidCredentialsforLogin(LoginPageTest.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="4.587">
    <skipped/>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="3.257">
    <failure message="Dashboard is not displayed expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Dashboard is not displayed expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at testScripts.LoginPageTest.verifyValidCredentialsforLogin(LoginPageTest.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="4.502">
    <failure message="Dashboard is not displayed expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Dashboard is not displayed expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at testScripts.LoginPageTest.verifyValidCredentialsforLogin(LoginPageTest.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="3.055">
    <skipped/>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="3.440">
    <skipped/>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="3.389">
    <skipped/>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="3.127">
    <failure message="Dashboard is not displayed expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Dashboard is not displayed expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at testScripts.LoginPageTest.verifyValidCredentialsforLogin(LoginPageTest.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="5.119"/>
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyInvalidPasswordforLogin" time="3.309"/>
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="4.456">
    <skipped/>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyInvalidUserNameforLogin" time="3.209"/>
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="4.615">
    <skipped/>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
  <testcase classname="testScripts.LoginPageTest" name="verifyValidCredentialsforLogin" time="3.118">
    <skipped/>
  </testcase> <!-- verifyValidCredentialsforLogin -->
  <system-out/>
</testsuite> <!-- testScripts.LoginPageTest -->
